loops_lab.js

```diff
@@ -0,0 +1,208 @@
+//   }
+//   else if (x%5 === 0 ) {
+//     console.log("Buzz")
+//   }
+//   else {
+//     console.log( x )
+//   }
+// }
+// ---------------------------------------------------------------------------------------------------------------------------------------------------
+// ### Bonus:
+//
+// 1. Write a program that would log the [lyrics of the song 99 Bottles of Beer](http://www.99-bottles-of-beer.net/lyrics.html). This is the first verse of the song:
+//
+//   ```
+//   99 bottles of beer on the wall, 99 bottles of beer.
+//   Take one down, pass it around, 98 bottles of beer on the wall.
+//   ```
+//   This verse is repeated, each time with one less bottle, until the number of bottles is 0. When the number of bottles is 2, the verse is:
+// ---------------------------------------------------------------------------------------------------------------------------------------------------
+//   ```
+//   2 bottles of beer on the wall, 2 bottles of beer.
+//   Take one down, pass it around, 1 bottle of beer on the wall.
+//   ```
+//
+//   In the last line, the word bottles (plural), is  replaced with bottle (singular)
+//
+//   When the number of bottles is 1, the verse is:
+//
+//   ```
+//   1 bottle of beer on the wall, 1 bottle of beer.
+//   Take one down, pass it around, No more bottle of beer on the wall.
+//   ```
+//
+// ___
+// ---------------------------------------------------------------------------------------------------------------------------------------------------
+// 2. Use the `assignGrade` function (given below). Write a function that logs each number from 60 - 100 along with its corresponding letter score.
+// Exp For each number from 81 to 90, log B, like so:
+//
+// ```js
+// 60 - F
+// ...
+// 81 - B
+// 82 - B
+// 83 - B
+// ...
+// 93 - A
+// 94 - A
+//
+// for(let x = 60; x <= 100; x++){
+//     if (x >= 91) {
+//         console.log(x + ' - A');
+//     } else if (x <= 90 && x >= 81) {
+//         console.log(x + ' - B');
+//     } else if (x < 81 && x >= 71) {
+//         console.log(x + ' - C');
+//     } else if (x < 71 && x >= 61) {
+//         console.log(x + ' - D');
+//     } else {
+//         console.log(x + ' - F');
+//     }
+//   }
+
+//   What is happening here? Can you explain it?
+// </details>
+//
+// ---------------------------------------------------------------------------------------------------------------------------------------------------
+// 3. Given an integer N draw a square of N x N asterisks. Look at the examples.
+//
+// Example 1:
+// Input: `let N = 2`
+//
+// Output:
+// ```js
+// **
+// **
+// let n = 2;
+// let string = '';
+//   for (let i = 0; i < n; i++){
+//   string += "*"
+// }
+// console.log(string)
+// for (let i = 1; i < n; i++){
+//   string += ""
+// }
+// console.log(string)
+// //
+// Example 2:
+// Input: `let N = 3`
+//
+//
+// Output:
+// ```js
+// ***
+// ***
+// ***
+// //---------------------------------------------------------
+// let n = 3;
```

Comment:
Maybe there can be a better way of doing this instead of having three for loops. If change the n to 100 then we would need 100 different for loops. Think of different way that we can solve this.